toucan_connectors/http_api/pagination_configs.py

```diff
@@ -56,6 +56,13 @@ class OffsetLimitPaginationConfig(PaginationConfig):
     offset: int = Field(0, **UI_HIDDEN)
     limit_name: str = "limit"
     limit: int
+    data_length_filter: str = Field(
+        ".",
+        description=(
+            "Filter to determine the data length and compare it to the limit value. "
+            "It must point to a list of results. " + FilterSchemaDescription
```

Comment:
```suggestion
            "Filter to access the received data. Allows to compare its length to the limit value. "
            "It must point to a list of results. " + FilterSchemaDescription
```

Wouldn't `data_filter` be a better name ?